master/setup.py

```diff
@@ -29,6 +29,10 @@
 from distutils.command.install_data import install_data
 from distutils.command.sdist import sdist
 
+if "bdist_wheel" in sys.argv:
+    NOT_WHEEL = False
+else:
+    NOT_WHEEL = True
```

Comment:
A boolean with "not" in the name is confusing!  Can you reverse this to "BUILDING_WHEEL" or something similar?